src/js/client.js

```diff
@@ -62,11 +61,13 @@ class Client {
 
   // Send a looking-for-game message.
   register(hasComputerOpponent) {
-    this.registered = true; // UI checks this to refresh on game seek
-    console.log("registering as " + this.name)
+    // UI checks this to refresh on game seek
+    this.registered = true; 
```

Comment:
i think it would be just a bit cleaner if you initialized this.registered to false in the constructor, and added a little comment there explaining what it meant (that at some point this client connected to the server? i'm not quite clear)